packages/marathon/buildinfo.json

```diff
@@ -2,8 +2,8 @@
   "requires": ["java"],
   "single_source": {
     "kind": "url_extract",
-    "url": "https://downloads.mesosphere.com/marathon/v1.3.6/marathon-1.3.6.tgz",
-    "sha1": "0d6a3f0abd4188cbb7d9d64e729f16489cb9bc6f"
+    "url": "http://downloads.mesosphere.io/marathon/v1.4.0-snap29/marathon-1.4.0-snap29.tgz",
```

Comment:
required: Https + downloads.mesosphere.com, part of the standard for all of these.